Menus/PauseMenuGUI.cpp

```diff
@@ -0,0 +1,325 @@
+			if (guiEvent.GetType() == GUIEvent::Command) {
+				if (guiEvent.GetControl() == m_PauseMenuButtons[PauseMenuButton::ResumeButton]) {
+					return true;
+				} else if (guiEvent.GetControl() == m_PauseMenuButtons[PauseMenuButton::SaveGameButton]) {
+					if (g_ActivityMan.GetActivityAllowsSaving() && g_ActivityMan.SaveCurrentGame("QuickSave")) {
+						g_GUISound.ConfirmSound()->Play();
+						return true;
+					} else {
+						g_GUISound.UserErrorSound()->Play();
+					}
+				} else if (guiEvent.GetControl() == m_PauseMenuButtons[PauseMenuButton::LoadLastSaveButton]) {
+					if (g_ActivityMan.LoadAndLaunchGame("QuickSave")) {
+						g_GUISound.ConfirmSound()->Play();
+						return true;
+					} else {
+						g_GUISound.UserErrorSound()->Play();
+					}
+				} else if (guiEvent.GetControl() == m_PauseMenuButtons[PauseMenuButton::SettingsButton]) {
+					SetActiveMenuScreen(PauseMenuScreen::SettingsScreen);
+				} else if (guiEvent.GetControl() == m_PauseMenuButtons[PauseMenuButton::ModManagerButton]) {
+					SetActiveMenuScreen(PauseMenuScreen::ModManagerScreen);
+				} else if (guiEvent.GetControl() == m_PauseMenuButtons[PauseMenuButton::BackToMainButton]) {
+					g_GUISound.BackButtonPressSound()->Play();
+					m_UpdateResult = PauseMenuUpdateResult::BackToMain;
+				}
+			}
+			if (guiEvent.GetType() == GUIEvent::Notification && (guiEvent.GetMsg() == GUIButton::Focused && dynamic_cast<GUIButton *>(guiEvent.GetControl()))) {
+				g_GUISound.SelectionChangeSound()->Play();
+			}
+		}
+		return false;
+	}
+
+/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+	void PauseMenuGUI::UpdateHoveredButton(const GUIButton *hoveredButton) {
+		int hoveredButtonIndex = -1;
+		if (hoveredButton) {
+			hoveredButtonIndex = std::distance(m_PauseMenuButtons.begin(), std::find(m_PauseMenuButtons.begin(), m_PauseMenuButtons.end(), hoveredButton));
+			if (hoveredButton != m_HoveredButton) {
+				m_PauseMenuButtons[hoveredButtonIndex]->SetText(m_ButtonHoveredText[hoveredButtonIndex]);
+			}
+			m_HoveredButton = m_PauseMenuButtons[hoveredButtonIndex];
+		}
+		if (!hoveredButton || hoveredButtonIndex != m_PrevHoveredButtonIndex) {
+			m_PauseMenuButtons[m_PrevHoveredButtonIndex]->SetText(m_ButtonUnhoveredText[m_PrevHoveredButtonIndex]);
+		}
+
+		if (hoveredButtonIndex >= 0) {
+			m_PrevHoveredButtonIndex = hoveredButtonIndex;
+		} else {
+			m_HoveredButton = nullptr;
+		}
+	}
+
+/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+	void PauseMenuGUI::BlinkResumeButton() {
+		if (m_HoveredButton && m_HoveredButton == m_PauseMenuButtons[PauseMenuButton::ResumeButton]) {
+			m_PauseMenuButtons[PauseMenuButton::ResumeButton]->SetText(m_ResumeButtonBlinkTimer.AlternateReal(500) ? m_ButtonHoveredText[PauseMenuButton::ResumeButton] : "]" + m_ButtonHoveredText[PauseMenuButton::ResumeButton] + "[");
```

Comment:
lmao `]` is capital font for `>`?
So hacky lol

Reply:
It is what it is \_/¯( ͜。 ͡ʖ ͜。)¯\\\_